### PR TITLE
Add widget for U-Report app stories

### PR DESCRIPTION
**Pull request by teehamaral**
This feature enables countries adding U-Report app stories in the website.

### CODE REVIEW

**Comment by nicpottier** [comment]
Hi there, can you tell us a bit about this feature, it is the first we hear about it and as far as we can tell the first from the UNICEF project managers for U-Report as well.

In general it is always best to start a discussion before opening a PR so we can talk through use cases and best approach, so we're not starting from a good base here, but let's start with what exactly this does. :)

**Comment by nicpottier** [comment]
Howdy, if you change this to make the ihlasoft URL configurable in the settings and add a unit test for the stories appearing when properly configured then we can merge this.

**Comment by teehamaral** [comment]
Hey @nicpottier, do you prefer that Ilhasoft URL is set to the value http://ureportapp.ilhasoft.mobi/widget/ or http://ureportapp.ilhasoft.mobi/ in settings file?

**Comment by nicpottier** [comment]
No preference, should just be something configurable.

**Comment by teehamaral** [comment]
OK.

**Comment by nicpottier** [comment]
test?

**Comment by teehamaral** [comment]
The test "test_set_story_widget_url" would not be necessary to test?

File: ureport/public/tests.py [hunk 1]
```diff
@@ -203,6 +203,14 @@ def test_is_rtl_org_processors(self):
         self.assertEquals(response.request['PATH_INFO'], '/')
         self.assertTrue(response.context['is_rtl_org'])
 
+    def test_set_story_widget_url(self):
+        import requests
+        story_widget_url = getattr(settings, 'STORY_WIDGET_URL', None)
+        iso_code = 'BRA'
+        full_url = "%s%s" % (story_widget_url, iso_code)
+        r = requests.get(full_url)
+        self.assertEqual(r.status_code, 200)
```
Comment (nicpottier) [inline ureport/public/tests.py]: This is testing a live URL in a unit test, definite no no. Use mock to properly mock the request.

You should be testing whether the widget is properly included on the index page when the appropriate settings are included for the org and settings file.

Comment (teehamaral) [inline ureport/public/tests.py]: Ok, I will correct. Thank you.

**Comment by nicpottier** [comment]
Looks ok to me.. @norkans7 merge when you have a chance.

**Comment by norkans7** [comment]
Yep

**Pull request merged**